src/model.rs

```diff
@@ -97,6 +97,8 @@ pub(crate) struct ComponentStatus {
 pub(crate) struct Status {
     /// Maps a component name to status
     pub(crate) components: BTreeMap<String, ComponentStatus>,
+    /// If a CoreOS system is detected, this will be the imgid from the "aleph version"
+    pub(crate) coreos_aleph_imgid: Option<String>,
```

Comment:
Assuming that in the future you want to implement "unknown" as described at the bottom of https://github.com/coreos/bootupd/issues/38, should this be something like `install_label: Option<Adopted>` where Adopted is an tagged-kind enum? Something like:
```
enum Adopted {
  CoreOSAleph(String),
  ManuallyForced(String),
  Unknown,
}
```

Reply:
I tweaked things to take a step in this direction.

Reply:
Ack, SGTM. Although I got slightly confused with the new patch. My understanding was that with this field we were trying to answer the question "what was the original version of the bootloader at install time" while now it seems to have morphed into "current OS auto-detection".
I guess I'll wait for the rest of the pile of patches to get a better grasp of the whole picture.